proposals/0000-automatically-derive-properties-for-enum-cases.md

```diff
@@ -74,10 +74,16 @@ Properties reduce code size and enhance readability. Contrast the following prop
 
 ``` swift
 // select items supporting associated value `value`
-array.filter { $0.value != nil }
+array.compactMap { $0.value }
 ```
 
-This approach is widely applicable. The Swift Standard Library provides many higher-order methods, including `map`, `filter`, `reduce`, `flatMap`, `sorted`, `first(where:)`, and more. Each is easier to use, read, and maintain when expressed as single lines.
+This approach is widely applicable. The Swift Standard Library provides many higher-order methods, including `map`, `filter`, `reduce`, `flatMap`, `compactMap`,  `sorted`, `first(where:)`, and more. Each is easier to use, read, and maintain when expressed as single lines.
+
+The benefit is arguably even more significant for APIs which provide KeyPath overloads (and more generally in the future should KeyPaths ever become a subtype of functions):
```

Comment:
Ha! It's almost as if you saw my other branch 😉

https://github.com/stephencelis/swift-evolution/blob/key-path-getter-promotion/proposals/0000-key-path-getter-promotion.md

Reply:
Awesome!  I didn't see that but this subtype relationship has been discussed in the past.